proto/rill/runtime/v1/api.proto

```diff
@@ -190,6 +190,11 @@ service RuntimeService {
   rpc ListConnectors(ListConnectorsRequest) returns (ListConnectorsResponse) {
     option (google.api.http) = {get: "/v1/connectors/meta"};
   }
+
+  // WatchFiles streams repo file update events. It is not supported on all backends.
```

Comment:
Can we move it up to after `ListFiles`? (and same with Request/Response messages for easy navigation